docker/nginx.conf.template

```diff
@@ -0,0 +1,25 @@
+server {
+  listen 8080;
+  server_name localhost;
+  root /usr/share/nginx/html/;
+  access_log off;
+  index index.html;
+
+  add_header 'Access-Control-Allow-Origin' '*' always;
+  expires 4h;
+
+  gzip on;
+
+  gzip_types application/javascript text/css application/json;
+
+  location / {
+    try_files $uri $uri/ =404;
+  }
+
+  location = / {
+    index $APP_PATH/index.html;
```

Comment:
Here we should also change to expiration (https://nginx.org/en/docs/http/ngx_http_headers_module.html)
```suggestion
    index $APP_PATH/index.html;
    expires -1:
```